src/sentry/snuba/spans_rpc.py

```diff
@@ -396,3 +398,55 @@ def _process_all_timeseries(
             result.sample_count[index][label] = data_point.sample_count
 
     return result
+
+
+def run_trace_query(
+    trace: str,
```

Comment:
nit: just because I found myself asking, is this trace_id or something else? 😅 
```suggestion
    trace_id: str,
```